src/react/components/pages/connections/connectionsPage.test.tsx

```diff
@@ -166,11 +162,9 @@ describe("Connections Page", () => {
 
             setImmediate(() => {
                 expect(saveConnectionSpy).toBeCalledWith(connection);
-                expect(assetProvider.initialize).toBeCalled();
```

Comment:
Any reason you are removing this?  @tbarlow12 recently added this since asset providers should be initialized while saving connections.

Reply:
No--I had removed it for testing. Adding back in.